packages/webapp/src/elections/components/ongoing-election-components/ongoing-round-segment.tsx

```diff
@@ -247,7 +243,9 @@ export const OngoingRoundSegment = ({
                         : "This round is finalizing. Please submit any outstanding votes now. You will be able to come back later to upload election videos if your video isn't ready yet."}
                 </Text>
             </Container>
-            {isVotingOpen ? (
+            {voteData?.length === 0 ? (
+                <div>you didn't participate in this round</div>
```

Comment:
In what event would this display? If it should _never_ display to the user, then we should consider dropping it. If it can display in some cases, let's capitalize it and wrap it properly in a `<Container />` so it presents acceptably.